packages/docregistry/src/context.ts

```diff
@@ -503,31 +502,36 @@ export class Context<
    * Handle an initial population.
    */
   private _populate(): Promise<void> {
```

Comment:
Mild suggestion (feel free to ignore) but it seems like this function would be easier to read if it were converted to `private async _populate(): Promise<void>`

Reply:
That's what I had before, but it was originally not async so I removed it to minimize changes. I can add it back though.